server/routes/api/v1/petTypesRouter.js

```diff
@@ -13,5 +15,16 @@ petTypesRouter.get("/", async (req, res) => {
         res.status(500).json({error: err});
     }
 })
+petTypesRouter.post("/", async (req,res)=> {
```

Comment:
Add a newline above and check your indentation settings -- it looks like your indentation might be set to 4 spaces!